openmdao/drivers/pyoptsparse_driver.py

```diff
@@ -207,6 +207,7 @@ def __init__(self, **kwargs):
         self.supports['multiple_objectives'] = True
         self.supports['two_sided_constraints'] = True
         self.supports['linear_constraints'] = True
+        self.supports['linear_only_designvars'] = True
```

Comment:
could I suggest adding a 'desc' to this option?  I know the others don't have it, but this one seems a bit subtle, having to do with the driver implementation ("doesn't split them internally "), and it may not be clear to users what it means...